gremlin-go/driver/performance-tests/java-test/src/main/java/PerformanceTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// CAUTION!!!
+// This file is intended for generating Performance test results only.
+// It is not to be used in a production setting.
+// It is being made available as a reference to the test results for
+// anyone that wants see the code used to obtain the performance metrics.
```

Comment:
```suggestion
// DO NOT USE - THIS WILL BE DELETED
// This file is intended for generating Performance test results only.
// It is not to be used in a production setting.
// It is being made available as a reference to the test results for
// anyone that wants see the code used to obtain the performance metrics.
```